v2.json

```diff
@@ -300,6 +300,14 @@
               "definition": "Activity types that fit under the 'Yoga' drop down filter on the Join the Movement virtual timetable",
               "beta:publisher": "Sport England",
               "inScheme": "https://openactive.io/activity-list"
+            },
+            {
+              "type": "beta:ConceptCollection",
+              "id": "https://bpa-parasport.github.io/parasport-openactive/collections/something-different.jsonld",
+              "prefLabel": "Something a little bit different",
```

Comment:
I have a slight hesitation around the `prefLabel` here, just seems a bit wordy. That might be fine, not sure

Reply:
Thanks for the feedback, @civsiv. @nickevansuk, do you have any thoughts on wordiness of the `prefLabel`?

Reply:
Assume this matches the branding on their side ("Something a little bit different"), fine if so?

Reply:
I believe so. Thanks for confirming.

Reply:
Approved then!